kubeflow/trainer/backends/kubernetes/utils.py

```diff
@@ -228,6 +235,20 @@ def get_resources_per_node(
     if "gpu" in resources:
         resources["nvidia.com/gpu"] = resources.pop("gpu")
 
+    # Optional alias for MIG: "mig-<profile>" -> "nvidia.com/mig-<profile>"
+    # Example: "mig-1g.5gb" -> "nvidia.com/mig-1g.5gb"
```

Comment:
Please can you update the docstring for CustomTrainer and CustomTrainerContainer to explain how to set `mig` resources: https://github.com/kubeflow/sdk/blob/main/kubeflow/trainer/types/types.py#L41 ?